### PR TITLE
Add common field problems article

### LINKED ISSUE #1795: Troubleshooting Idle Robot on the Field
I want a better name for this.  This should all be information that a CSA / FTA knows but this will help let teams inside our head and let us know what we are looking for when a robot goes idle on the field and what additional troubleshooting should be done based on this problem discovery.

### PR DESCRIPTION
**Pull request by Daltz333**
Closes #1795 

Comments and discussion are welcome. Additionally, I'd love if people can share DS logs of various problems that they encountered *while on the field*. Lots of code related issues that are identified at compile time are not usually present when on the field.

### CODE REVIEW

**Review by sciencewhiz — COMMENTED** [review]
I would also add best practices to keep field problems from happening

get to the field on practice day, even if your robot isn't done
test code after deploying
run practice mode in pits

I would also add loose wires causing roborio/radio reboots, and can link to driver station log article.

File: source/docs/software/troubleshooting/common-field-problems.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+.. important:: Remember to never eliminate any possibility! It never hurts to double or even triple check that everything is working properly.
+
+Robot is stuttering and the RSL lights are dimming
+--------------------------------------------------
+
+Whenever your robot seems to give jerking motions and the RSL lights are dimming, this is usually a sign of :doc:`brownouts </docs/software/roborio-info/roborio-brownouts>`. One of the first steps you can take to resolving a brownout is identify when it occurred and any notable correlating events. Did you go into a match with your battery too low? Are you drawing too much current somehow? Can you reproduce this in the pit?
+
+One of the most useful tools for identifying brownout causes is the :doc:`driver station log viewer </docs/software/driverstation/driver-station-log-viewer>`.
+
+.. todo:: add examples showcasing a brownout
```
Comment (sciencewhiz) [inline source/docs/software/troubleshooting/common-field-problems.rst:15]: There's an example in the brownouts article

**Comment by gerth2** [comment]
Just found this while staring at the other troubleshooting issues. I'll merge the content into my PR. If this one goes first that's still fine, the content should be preserved as I add on more stuff and reorganize.

**Comment by Daltz333** [comment]
Closing this in favor of #1884

**Pull request closed without merging**